pkg/slo-controller/noderesource/noderesource_controller_test.go

```diff
@@ -197,20 +198,15 @@ func Test_NodeResourceController_ColocationEnabled(t *testing.T) {
 	assert.Equal(t, int64(65000), batchCPU)
 
 	// reset node resources
-	r.Clock = clock.NewFakeClock(r.Clock.Now().Add(time.Hour))
```

Comment:
will `isDegradeNeeded` be covered after this fake clock got deleted?

Reply:
never mind. I saw you add a test for `isDegradeNeeded` exclusively in `pkg/slo-controller/noderesource/plugins/batchresource/plugin_test.go`